patroni/daemon.py

```diff
@@ -134,6 +134,11 @@ def run(self) -> None:
         Start the logger thread and keep running execution cycles until a SIGTERM is eventually received. Also reload
         configuration upon receiving SIGHUP.
         """
+        try:
+            from systemd import daemon
+            daemon.notify("READY=1")
+        except ImportError:
+            logger.warn("Systemd integration is not supported")
```

Comment:
I am not sure if it should be a warning.
IMO, info is enough.

Reply:
Done !